orbs/phraseapp/README.md

```diff
@@ -0,0 +1,87 @@
+# Phraseapp Orb
+
+## Proposal
+This orb was created to make the merge of the changes made in the app.phrase with the GitHub project repository simple.
+
+## Configuration Needed
+1) Add [pharaseapp.yml](.phraseapp.sample.yml) to the root of your project.
+It will be used to know what locales we need to fetch from the app.phrase, the path to pull the changes and the format of the configuration file.
+```
+phraseapp:
+  available_locales:
+    en
+    fr
+    de
+  push:
+    sources:
+      - file: ./config/locales/<locale_name>.yml
+        params:
+          file_format: yml
+  pull:
+    targets:
+      - file: ./config/locales/<locale_name>.yml
+        params:
+          file_format: yml
+```
+
+
+2) Add jobtomate as a user to the project collaborators in the github repository. Instructions could be found [here](https://github.com/jobteaser/<repository>/settings/access).  
+
+
+3) Add the following environment variables to the circle. The path to add it could be found [here](https://app.circleci.com/settings/project/github/jobteaser/<repository>/environment-variables). The token for the project can be generated [here](https://app.phrase.com/settings/oauth_access_tokens). 
+- PHRASEAPP_ACCESS_TOKEN 
+- PHRASEAPP_PROJECT_ID
+
+
+4) Update the .circleci/config: 
+
+```
+orbs:
+  service: "jobteaser/service@0.10.0"
+  phraseapp: "jobteaser/phraseapp@dev:0.0.0"
+
+#if you want to run the automerge on the build stef of the main workflow
+workflows:
+  main 
+    jobs:
+      - phraseapp/automerge-phraseapp-pr:
+      name: "automerge-phraseapp-pr"
+      context: "build"
+      requires: ["test"]
+      filters:
+        branches:
+          only: phraseapp-locales
```

Comment:
**question**

Is the indentation correct here ? Shouldn't it be indented 4 spaces to the right ? (I am always at loss regarding yaml indentation 😅 so maybe it correct ?)